src/libstrongswan/plugins/openssl/openssl_plugin.c

```diff
@@ -549,6 +550,13 @@ METHOD(plugin_t, get_features, int,
 			PLUGIN_PROVIDE(KE, MODP_768_BIT),
 			PLUGIN_PROVIDE(KE, MODP_CUSTOM),
 #endif
+#ifdef OPENSSL_IS_AWSLC
```

Comment:
Is there a specific `OPENSSL_NO_*` constant for ML-KEM? Or is AWS-LC always built with ML-KEM (in versions that support it)?

Reply:
It's the latter case. Our consumers tend to use either compile time checks or logic around the version number macro to figure out exactly which features are available. I can add a version check here, but since it's still early days with AWS-LC in this code base, I think we can still get away with avoiding it.